code/modules/mob/living/carbon/human/species.dm

```diff
@@ -298,8 +298,11 @@
 	/*abilities = list(/client/proc/test_ability)*/
 
 /datum/species/human/vatborn
-	name = "Vatborn"
-	name_plural = "Vatborns"
+	name = "Vat-Born Human"
+	name_plural = "Vat-Born Humans"
+	icobase = 'icons/mob/human_races/r_vatborn.dmi'
+	deform = 'icons/mob/human_races/r_vatborn.dmi'
+	species_flags = HAS_SKIN_TONE|HAS_LIPS|HAS_UNDERWEAR
```

Comment:
I wouldnt be against making vatborn have the yellow human sprites and the vatgrown having the pale sprites or the other way around

Would help with visual differentiation

Reply:
@XSlayer300 ?